intermine/intermine
# Footer fix issue for #1604

## Details

Issue #1604 - looked like some kind of regression/merge issue with footer. 

* Fixed missing images by cherry-picking my old commits.
* added apple ios store link
* ensure that if mines don't have the "credits" variable set, the screen won't just say ???credits (won't display anything if variable unset).

## Testing

1. deploy local instances of humanmine / flymine.
2. view footer. Are there broken images? See #1604 for the "bad" scenario that should be fixed.

## Checklist

Before your pull request can be approved, be sure to check all boxes:

~- [ ] Passing unit test for new or updated code (if applicable)~
- [x] Passes all tests – according to Travis
~- [ ] Documentation (if applicable)~
- [x] Single purpose
- [x] Well commented code
- [ ] Checkstyle
